core/src/java_interface.rs

```diff
@@ -407,6 +410,24 @@ pub extern "system" fn Java_app_lockbook_core_CoreKt_createFile(
     )
 }
 
+#[no_mangle]
+pub extern "system" fn Java_app_lockbook_core_CoreKt_getLastUpdated(
```

Comment:
You should call this something like `convertToHumanDuration` or something like that